test/index.js

```diff
@@ -240,14 +320,22 @@ describe('normalize.undefined', function() {
     expect(result).toEqual(value);
     done();
   });
+
+  it('rejects values that won\'t coerce to undefined', function(done) {
+    var value = 'invalid';
+    var result = normalize.boolean(value);
```

Comment:
I think this should be `normalize.function(value)`.  Probably copy-paste oversight.